File: dev/20190702/k2works/api/rental/src/main/java/rental/application/StatementService.java
package rental.application;

import org.springframework.stereotype.Service;
import rental.domain.Customer;
import rental.domain.Movie;
import rental.domain.Rental;

@Service
public class StatementService {

  public Customer createStatment() {
    Movie newReleaseMovie = new Movie("新作", Movie.NEW_RELEASE);
    Movie childrenMovie = new Movie("子供", Movie.CHILDREN);
    Movie regularMovie = new Movie("一般", Movie.REGULAR);

    Rental newRelease = new Rental(newReleaseMovie, 3);
    Rental children = new Rental(childrenMovie, 2);
    Rental regular = new Rental(regularMovie, 1);

    Customer customer = new Customer("山田");
    customer.addRental(newRelease);
    customer.addRental(children);
    customer.addRental(regular);

    return customer;
  }
}

File: dev/20190710/k2works/api/video_rental/statement_service.py
from customer import Customer
from movie import Movie
from rental import Rental


class StatementService:
    def create_text_statement(self):
        customer = self.create_statement()
        return customer.statement()

    def create_html_statement(self):
        customer = self.create_statement()
        return customer.html_statement()

    def create_statement(self):
        new_release_movie = Movie("新作", Movie.NEW_RELEASE)

        children_movie = Movie("子供", Movie.CHILDREN)

        regular_movie = Movie("一般", Movie.REGULAR)

        new_release = Rental(new_release_movie, 3)

        children = Rental(children_movie, 2)

        regular = Rental(regular_movie, 1)

        customer = Customer("山田")

        customer.add_rental(new_release)

        customer.add_rental(children)

        customer.add_rental(regular)

        return customer
